kalagraphs.module

```diff
@@ -154,10 +188,13 @@ function kalagraphs_theme() {
     }
 
     $item = [
-      'template' => $component,
-      'path' => "{$info['path']}/$component",
+      'template' => $info['template'],
+      'path' => "{$info['path']}/{$info['template']}",
       // See _kalagraphs_populate_fields() for explanation of this necessity.
-      'preprocess functions' => ['_kalagraphs_populate_fields'],
+      'preprocess functions' => [
+        '_kalagraphs_populate_fields',
+        "template_preprocess_kalagraphs_$component",
```

Comment:
@gremy Pointed out that before we can make this work, we would need a generic "base hook" whose preprocess functions get called in addition to the more specific theme implementation hooks.